### ISSUE TITLE
Cannot read property 'getBoundingClientRect' of null

### ISSUE THREAD

**Issue by che3vinci** [quilljs/quill#969]
[Describe the issue]
line 9420：var containerBounds = this.boundsContainer.getBoundingClientRect();

Uncaught TypeError: Cannot read property 'getBoundingClientRect' of null

the source code

 ```
<script src="http://cdn.quilljs.com/1.0.3/quill.js"></script>

   var quill = new Quill('#editor', {
        modules: {
            toolbar: [
                [{ header: [1, 2, false] }],
                ['bold', 'italic', 'underline'],
                ['image', 'code-block']
            ]
        },
        placeholder: 'Compose an epic...',
        theme: 'bubble' // or 'bubble'
    });

```
**Expected behavior**:

**Actual behavior**:

**Platforms**:
mac osx

Include browser, operating system and respective versions
chrome version 52.0.2743.116 (64-bit)

**Version**:

Run `Quill.version` to find out
1.0.3

**Comment by benbro** [comment]
Does adding "bounds: document.body," fix it?

```javascript
var quill = new Quill('#editor', {
	modules: {
		toolbar: [
			[{ header: [1, 2, false] }],
			['bold', 'italic', 'underline'],
			['image', 'code-block']
		]
	},
	bounds: document.body,
	placeholder: 'Compose an epic...',
	theme: 'bubble' // or 'bubble'
});
```
This issue will be fixed in the next version:
https://github.com/quilljs/quill/pull/944

**Comment by benbro** [comment]
Should be fixed in 1.0.4

**Comment by lucasocon** [comment]
I got the same error with 1.2.6 version required from CDN.
and solved adding:
` bounds: document.body`
